README.md

```diff
@@ -1,2 +1,97 @@
-# cp2-flask-bucketlist-api
-Checkpoint 2 project on creating a RESTful bucketlist api using python flask.
+# Flask Bucketlist API
+[![Build Status](https://travis-ci.org/andela-austinroy/cp2-flask-bucketlist-api.svg?branch=develop)](https://travis-ci.org/andela-austinroy/cp2-flask-bucketlist-api) [![Code Climate](https://codeclimate.com/github/andela-austinroy/cp2-flask-bucketlist-api/badges/gpa.svg)](https://codeclimate.com/github/andela-austinroy/cp2-flask-bucketlist-api) [![Code Health](https://landscape.io/github/andela-austinroy/cp2-flask-bucketlist-api/develop/landscape.svg?style=flat)](https://landscape.io/github/andela-austinroy/cp2-flask-bucketlist-api/develop) [![license](https://img.shields.io/github/license/andela-austinroy/cp2-flask-bucketlist-api.svg)]()
+
+Checkpoint 2 project on creating a RESTful bucketlist API using python flask. This project makes use of the Flask framework to execute a RESTful API that allows users to create edit and save bucketlists on a database. Response data is in json format.
+
+## Getting Started
+
+These instructions will get you a copy of the project up and running on your local machine for development and testing purposes.
+
+### Prerequisites
+
+1. Python 2.7
+Comes inbuilt for unix but can also be downloaded from
+
+```
+https://www.python.org/downloads/
```

Comment:
It's thoughtful of you to include this url. Have you considered making it a hyperlink instead of just formatted code? Like [Python Language Website Downloads](https://www.python.org/downloads/)